covid19-dashboard/src/DataTable.tsx

```diff
@@ -210,8 +225,21 @@ export default class DataTable extends React.Component<
       );
     });
 
+    const content = this.state.categories.map(obj => {
+      return {id: obj.id, title: obj.title, enabled: obj.enabled}
+    })
+
     return (
       <>
+        <HeaderSelector content={content} onClick={(id: string) => {
+          const obj = this.state.categories.find(obj => obj.id === id)
+          if (obj) {
+            const enabled = localStorage[obj.id] === 'true'
+            obj.enabled = !enabled
+            localStorage[obj.id] = !enabled
+          }
+          this.forceUpdate()
```

Comment:
is it necessary to force an update? can this be managed by updating state instead?

Reply:
it was giving me issues, i'm working on the new design for the button selection as per your feedback.
I think we can leave it as is and it will change with the new design.

Reply:
that sounds reasonable. could you add a todo?